docs/sdk/push-notifications/fragments/ios/getting-started.md

```diff
@@ -1,168 +1,181 @@
 Enable your users to receive mobile push messages sent from the Apple (APNs) and Google (FCM/GCM) platforms. The Amplify CLI deploys your push notification backend using [Amazon Pinpoint](http://docs.aws.amazon.com/pinpoint/latest/developerguide/).
 You can also create Amazon Pinpoint campaigns that tie user behavior to push or other forms of messaging.
 
-## Pre-requisite
+## Set Up Your Backend
+
+### Prerequisites
 
 - Enrolled in the Apple Developer Program
 - Actual device to run the app on
 
+### Step-by-step
+
 1. Complete the [Get Started](~/start/start.md) steps before you proceed.
 
-2. Under the target, select "Signing & Capabilities", make sure the `Bundle Identifier` is unique. Make sure `Automatically manage signing` is enabled and your apple developer team is chosen.
+1. Under the target, select "Signing & Capabilities", make sure the `Bundle Identifier` is unique. Make sure `Automatically manage signing` is enabled and your apple developer team is chosen.
 
-3. Plug in the device and make sure it is registered and there are no errors on this page provisioning the profile.
+1. Plug in the device and make sure it is registered and there are no errors on this page provisioning the profile.
 
-4. On this page, click on `+ Capability`, and add `Push Notification` and `Background Modes`. For `Background Modes`, have `Remote notifications` checked.
+1. On this page, click on `+ Capability`, and add `Push Notification` and `Background Modes`. For `Background Modes`, have `Remote notifications` checked.
 
 ## Set Up Your Backend
```

Comment:
why is there two "setup your backend" H2s

Reply:
good catch, fixed it.